src/main/java/org/arl/fjage/remote/ArrayAdapterFactory.java

```diff
@@ -26,13 +26,16 @@
 class ArrayAdapterFactory implements TypeAdapterFactory {
 
   private boolean bare;
+  private int threshold;
```

Comment:
This whole file is also part of another update and should not be in this PR

Reply:
As the changes in https://github.com/org-arl/fjage/pull/212 and the rewrite-handler were required for testing, this branch was rebased onto the changes in https://github.com/org-arl/fjage/pull/212. The PR #212 has been merged and this PR currently has only rewrite-handler related changes.